### ISSUE TITLE
Wrong information in forum thread

### ISSUE THREAD

**Issue by mmermerkaya** [BobPalmer/USI-LS#108]
In the [release thread](http://forum.kerbalspaceprogram.com/index.php?/topic/105202-11-usi-life-support-040-more-bug-fixes/) you wrote:

> Orange suited Kerbals (Jeb, Bill, Bob, and Val) are immune to the ill effects of life support. They will still consume supplies if given, but do not leave the job, because they are just that awesome.

I think this was how you first implemented it, but after Jeb refused to work in my game, I checked the code and the default configs and realized that this is not the case anymore.

Thanks!

**Comment by TexasDex** [comment]
Yeah, this change just killed me.  Or killed Bob Kerman at least.
